integration_test/tests/query_tests.rs

```diff
@@ -1,5 +1,6 @@
 mod common;
 
+#[cfg(target_os = "linux")]
```

Comment:
This test suite routinely segfaults on Windows. Attempting to troubleshoot this with @pmeredit led us to deciding to disable the test for Windows for now.